caffe2/version_script.lds

```diff
@@ -2,6 +2,5 @@ pytorch {
   global:
     *;
   local:
-    _ZN4llvm*;
-    _ZNK4llvm*;
+    *4llvm*;
```

Comment:
is "4" intentionally included?

Reply:
Yup, the mangling process prepends the length of the namespace.  This new pattern doesn't exactly fill me with joy since someone *could* put `4llvm` in the name,  but it seems less likely than `llvm`.